### PR TITLE
Meeting notes 2020-01-21

### PR DESCRIPTION
**Pull request by chartgerink**
Metamorphosis of notes into minutes happens when you eat 🍝 and the reviewers approve

### CODE REVIEW

File: 20200114.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+-   Attending
```
Comment (dpaez) [inline 20200114.md]: I think this file was already merged (with some updates)

File: 20200121.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+-   Attending
+    - Julian Gruber [@juliangruber](https://twitter.com/juliangruber)
+    - Diego Paez [@carax](https://twitter.com/carax)
+    - Chris Hartgerink [@chartgerink](https://twitter.com/chartgerink)
+    - James Lomas
+-   [`@p2pcommons/sdk-js`](https://github.com/p2pcommons/sdk-js)
+    - Update by Diego
+      - Debugging SDK using two instances of the SDKs
+      - switching how we're managing namespaces for key management (?)
```
Comment (dpaez) [inline 20200121.md]:
```suggestion
      - switching how we're managing hyperdrives. Currently we are moving to using only one [corestore](https://github.com/andrewosh/corestore) with multiple `namespaces` and leaving behind passing the keypair ({publicKey, secretKey}).
```

File: 20200121.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+-   Attending
```
Comment (psobrakseaton) [inline 20200121.md]: `- Patch Sobrak-Seaton [@psobrakseaton](twitter.com/psobrakseaton)`

Comment (juliangruber) [reply 20200121.md]:
```suggestion
-   Attending
    - Patch Sobrak-Seaton [@psobrakseaton](twitter.com/psobrakseaton)
```

Comment (juliangruber) [reply 20200121.md]: Screenshot 2020-01-22 at 08 39 20

you can suggest these directly

File: 20200121.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      - JG: Can we do user testing?
+      	- Yes :) We could do some user interviews in the next weeks as well.
+-   [`@hypergraph-xyz/vault`](https://github.com/hypergraph-xyz/vault)
+    - Update by Julian
+      - Deployment is becoming more stable
+      	- Stripe webhook is setup and configured
+	- Authentication is implemented with magic links
+	- Next steps:
+	  - Stripe forms
+	  - Basic server side templating
```
Comment (juliangruber) [inline 20200121.md]: all of these should be indented one less, since they are not about the deployment stability

Comment (juliangruber) [reply 20200121.md]: basic server side templating has been implemented

File: 20200121.md (hunk 3 above)
Comment (psobrakseaton) [reply 20200121.md]: Thanks for pointing this out @juliangruber! I'll utilize the suggested edit function with the next critique.

**Pull request merged**